<!-- Ai -->
<!-- Do not remove any tags, just edit the text in between -->

<div class="row">
  <!-- Direction 1 -->
  <h6>Combine time-resolved single cell genomics with differentiable machine learning approaches</h6>

  <div class="col-sm-12 col-md-2">
    <a href="https://www.sciencedirect.com/science/article/pii/S0092867421015774">
<!--           <img
        id="gi-img"
        class="img-fluid my-2"
        src="assets/images/papers/stereo_seq.jpeg"
        alt="Stereo-seq cover paper"
      /> -->
      <img src="/assets/animations/hsc_fate.gif" alt="Vector field animations" style="width: 220px; height: 225px;">
    </a>
  </div>
<!--       <div class="col-3">
            <img src="/assets/animations/hsc_fate.gif" style="width: 220px; height: 225px;">
        </div>
        <div class="col-3">
            <img src="/assets/animations/hsc_fate_p.gif" style="width: 220px; height: 225px;">
        </div> -->

  <div class="col-sm-12 col-md-10 mb-sm-4">

    <p>Emergent single-cell genomics have enabled profiling of cell-state transitions with unprecedented scale. However, due to their destructive nature, it is generally infeasible to follow the same cell over time. Advances in single-cell profiling have fueled the development of computational approaches for inferring cellular dynamics from snapshot measurements, such as pseudotime and RNA velocity approaches. Furthermore, exciting developments in metabolic labeling enabled scRNA-seq approaches now enable us to obtain time-resolved transcriptomic kinetics by directly measuring “new” and “old” RNAs in a controllable manner. We recently developed the <a href="https://github.com/aristoteleo/dynamo-release">Dynamo</a> framework that overcomes fundamental limitations of conventional splicing-based RNA velocity analyses to enable accurate velocity estimations for time-resolved scRNA-seq datasets. Furthermore, we go beyond the discrete RNA velocity vectors to a continuous function that can be used to perform higher-order differentials to gain functional biological insights. Dynamo also establishes itself as one of the first tools to make non-trivial predictions of optimal reprogramming paths and <i>in silico</i> perturbation predictions with single cell datasets. We are continuing this line of research in the lab. </p>
  </div>

  <p>Major questions we are actively asking  in the lab include: </p>
    <div class="col-sm-12 col-md-10 mb-sm-4">
      <li>How to advance Dynamo’s analytical vector field approach to differentiable deep learning frameworks to dissect mechanisms of mammalian cell evolution, differentiation, maintenance, aging, and reprogramming? </li>
      <li>How to integrate RNA metabolic labeling with multi-omics experimentally, and then extend the RNA velocity approaches to multi-modal velocity of epigenetics, RNA, and protein? </li>
      <li>How to use methods like machine translation to translate single-cell gene expression states to other modalities, such as chromatin accessibility or proteomics, and <i>vice versa</i>?</li>
      <li>How to harmonize short-term RNA velocities with long-term cell state transitions measured with CRISPR-Cas9 based or mitochondria mutation-based lineage tracing?</li>
    </div>
    <div class="col-sm-12 col-md-2">
      <a href="https://www.sciencedirect.com/science/article/pii/S0092867421015774">
<!--           <img
          id="gi-img"
          class="img-fluid my-2"
          src="assets/images/papers/stereo_seq.jpeg"
          alt="Stereo-seq cover paper"
        /> -->
        <img src="/assets/animations/hsc_fate_p.gif" alt="In silico perturbation based on vector field" style="width: 220px; height: 225px;">
      </a>
    </div>