<!-- Ai -->
<!-- Do not remove any tags, just edit the text in between -->

<h6>Leverage spatial genomics to model cell-cell interaction and tissue organization in 3D space</h6>

<div class="col-sm-12 col-md-2">
  <a href="https://www.cell.com/cell/fulltext/S0092-8674(22)00399-3?_returnURL=https://linkinghub.elsevier.com/retrieve/pii/S0092867422003993?showall=true">
    <img
      id="gi-img"
      class="img-fluid my-2"
      src="assets/images/papers/stereo_seq.jpeg"
      alt="Stereo-seq cover paper"
    />
  </a>
</div>
<div class="col-sm-12 col-md-10 mb-sm-4">
  <p>Our Dynamo approach for RNA velocity vector field allows us to gain mechanistic insight into cell state transitions from the temporal axis. However, complex cell fate changes such as embryogenesis are not only temporally controlled but also spatially controlled. Unfortunately, routine single-cell approaches dissociate the cells, resulting in complete loss of spatial information. In order to best model the spatial axis, similar to the Dynamo work, we will apply, optimize, and develop novel technologies, such as Stereo-seq, STARmap and PIXEL-seq, that can generate the ideal spatial data. Although these promising technologies now allow us to profile cell states across subcellular, cellular, tissue, organ, and even whole embryo levels, analytical tools that fully leverage such data remain lacking. We recently developed a general analytical framework, <a href="https://github.com/aristoteleo/spateo-release">Spateo</a>, to model spatial transcriptomics at a multidimensional scale, ranging from single-cell segmentation, spatial domain clustering and digitization, cell-cell interactions, to whole organ and tissue 3D morphometric analysis. We will also continue this line of research in the lab.</p>
</div>

<p>Major questions we are actively asking in the lab include: </p>
<div class="col-sm-12 col-md-10 mb-sm-4">
  <li>How can we model RNA subcellular distribution and co-localization of image-based spatial transcriptomics using advanced machine learning techniques such as graph neural networks?</li>
  <li>How can we properly reconstruct the 3D volumetric structure of whole organs and embryos with serial profiling of spatial transcriptomics using optimal transport and more generalizable and efficient alternatives?</li>
  <li>How to enable novel 3D analysis of the reconstructed 3D models with 3D computer vision approaches?</li>
  <li>How to model cell-cell communication at cellular, tissue, and organ levels and in 3D space using spatially aware machine learning models?</li>
  <li>How can we extend spatial transcriptomics with multi-omics and Perturb-seq to build spatially resolved predictive models for these datasets?</li>
  </div>
<div class="col-sm-12 col-md-2">
    <a href="https://www.cell.com/cell/fulltext/S0092-8674(22)00399-3?_returnURL=https://linkinghub.elsevier.com/retrieve/pii/S0092867422003993?showall=true">
      <img src="/assets/animations/E7-9h_merged_completed_model_animation.gif" alt="E7-9h_merged_completed_model_animation" style="width: 200px; height: auto;">
    </a>
  </div>