<!-- Ai -->
<!-- Do not remove any tags, just edit the text in between -->

<h6>Moving towards cross-species foundational models of single cells</h6>
<div class="col-sm-12 col-md-1">
  <a href="https://www.cell.com/cell/fulltext/S0092-8674(22)00399-3?_returnURL=https://linkinghub.elsevier.com/retrieve/pii/S0092867422003993?showall=true">
    <img
      id="gi-img"
      class="img-fluid my-2"
      src="assets/images/foundation_model.png"
      alt="Stereo-seq cover paper"
    />
  </a>
</div>
<div class="col-sm-12 col-md-11 mb-sm-4">
  <p>Recent advances in machine learning, particularly with powerful, versatile foundational models like ChatGPT, have transformed various fields. These models use attention based transformer and are trained on extensive datasets, enabling them to perform exceptionally well in areas like image, text, and video processing with limited data. Advances in single-cell and spatial genomics has accumulated  massive datasets with tens of millions of single cells that provided necessary data to power up foundational models. While there have been several exciting attempts in building these models, such as Geneformer, scGPT, scFoundation and others, in biology with specific downstream tasks like batch effect removal and gene regulatory network analysis, their predictive capabilities are still lacking. To fulfill these unmet gaps, we have formed a vibrant team to actively investigating these possibilities.</p>
</div>

<div class="col-sm-12 col-md-8 mb-sm-4">
  <p>We are pursuing the following key questions:</p>
  <li>How to overcome the limitations of treating genes as token and cells as sentences in Geneformer, scGPT, scFoundation and others when building the single cell foundational model?</li>
  <li>How to extend existing foundational model to multi-modal datasets using machine translation, tabular learning and others?</li>
  <li>How to build cross-species foundational models so that we can unify the gene and cell embeddings across evolutionary scales? </li>
  <li>How to utilize spatiotemporal transformers to model the time-resolved and spatially-resolved single cell genomics, and in 3D space? </li>
  <li>How to perform non-trivial zero-shot downstream predictions, such as <i>in silico</i> perturbations and others, with the foundational model?  </li>
  <li>How to model cell growth, division, migration and others to simulate a differentiable <i>in silico</i> model of mamalian embryogenesis and organogenesis?  </li>
</div>
