<!-- C -->
<!-- Do not remove any tags, just edit the text in between -->

<p>
</p>
The Qiu Lab has developed a suite of computational software for preprocessing, analyzing, and interpreting emergent single-cell and spatial genomics data. During Dr. Qiu's Ph.D. at the University of Washington, under the guidance of Dr. Cole Trapnell, he developed <a href="https://github.com/cole-trapnell-lab/monocle-release">Monocle 2/3</a>, which robustly and accurately infers developmental trajectories from single-cell genomics data. In collaboration with Sreeram Kannan and Arman Rahimzamani at UW, he developed <a href="https://github.com/aristoteleo/Scribe-py">Scribe</a>, a tool for learning causal regulatory networks. As a postdoctoral fellow with Dr. Jonathan Weissman at the Whitehead Institute and MIT, together Yan Zhang from Jianhua Xingâ€™s lab at Pitt, he created <a href="https://github.com/aristoteleo/dynamo-release">Dynamo</a>, a predictive tool capable of making non-trivial predictions about the optimal set of reprogramming factors and performing <i>in silico</i> perturbations. He also worked with Joseph Min to develop <a href="https://github.com/aristoteleo/dynast-release">dynast</a>, an inclusive and efficient quantification tookit for metabolically labeled enabled scRNA-seq datasets. 
<p>
</p>
<p>
Currently, Dr. Qiu is leading a dynamic team in developing the <a href="https://github.com/aristoteleo/spateo-release">Spateo</a> framework, which facilitates multi-scale (subcellular to whole-embryo) modeling of emerging spatial genomics datasets. The Qiu lab is also actively engaged in creating differentiable models that map cell state transitions over time and space using graph neural networks, neuralODEs and others. Moreover, the Qiu Lab is committed to constructing foundational models of single cells using attention-based transformers and diffusion models.
</p>
<p>
Ultimately, the Qiu lab aims to develop an end-to-end, predictive foundational software ecosystem, <a href="https://github.com/aristoteleo/">Aristotle</a>, for quantitative spatiotemporal modeling of spatial and single-cell multiomics.
</p>
