<!-- Ai -->
<!-- Do not remove any tags, just edit the text in between -->

<!-- background point 1 -->
<div class="row">
  <div class="col-sm-12 col-md-1">
    <a href="https://www.cell.com/cell/fulltext/S0092-8674(22)00399-3?_returnURL=https://linkinghub.elsevier.com/retrieve/pii/S0092867422003993?showall=true">
      <img
        id="gi-img"
        class="img-fluid my-2"
        src="assets/images/papers/stereo_seq.jpeg"
        alt="Stereo-seq cover paper"
      />
    </a>
  </div>
  <div class="col-sm-12 col-md-11 mb-sm-4">
    <h6>Thanks to the recent progress in single-cell and spatial genomics, we are now able to profile transcriptomes and other “omics” from millions of cells.</h6>
    <p style="font-size: 14px;">On the cover: Stereo-seq dissects cell-type composition in an E16.5 mouse embryo sagittal section. In this issue, Chen et al. combined DNA nanoball (DNB)-patterned arrays and in situ RNA capture to create spatial enhanced resolution omics sequencing (Stereo-seq), which was applied to generate the Mouse Organogenesis Spatiotemporal Transcriptomic Atlas (MOSTA) with high sensitivity at single-cell resolution. This cover is taken from one of the representative single-cell segmentation results in a developing E16.5 mouse embryo section, and each color represents a cell type.</p>

  </div>
</div>

<!-- background point 2 -->
<div class="row">
  <div class="col-sm-12 col-md-1">
    <a href="https://www.nature.com/articles/s41586-021-03819-2">
      <img
        id="gi-img"
        class="img-fluid my-2"
        src="assets/images/papers/alpha_fold2.jpeg"
        alt="Stereo-seq cover paper"
      />
    </a>
  </div>
  <div class="col-sm-12 col-md-11 mb-sm-4">
    <h6>In parallel, the world has also witnessed breakthroughs in machine learning, epitomized by AlphaFold, ChatGPT and others. Thus the integration of genomics with machine learning has enormous potential.</h6>

    <p style="font-size: 11px;">On the cover: Proteins are essential to life, and understanding their 3D structure is key to unpicking their function. To date, only 17% of the human proteome is covered by an experimentally determined structure. Two papers in this week’s issue dramatically expand our structural understanding of proteins. Researchers at DeepMind, Google’s London-based sister company, present the latest version of their AlphaFold neural network. Using an entirely new architecture informed by intuitions about protein physics and geometry, it makes highly accurate structure predictions, and was recognized at the 14th Critical Assessment of Techniques for Protein Structure Prediction last December as a solution to the long-standing problem of protein-structure prediction. The team applied AlphaFold to 20,296 proteins, representing 98.5% of the human proteome. The predictions have been made freely available in partnership with the European Bioinformatics Institute, along with additional predictions for long human proteins and for 20 other model organisms. <strong>Work from DeepMind</strong>. </p>

  </div>
</div>

<!-- background point 3 -->
<div class="row">
  <div class="col-sm-12 col-md-1">
    <a href="https://www.sciencedirect.com/science/article/pii/S0092867421015774">
      <img
        id="gi-img"
        class="img-fluid my-2"
        src="assets/images/papers/dynamo.png"
        alt="Stereo-seq cover paper"
      />
    </a>
  </div>

  <div class="col-sm-12 col-md-11 mb-sm-4">
    <h6>To date, however, routine practices to explore single cell datasets in the field of computational biology have been largely descriptive, rather than predictive. The Qiu lab aims to pioneer in predictive models in single cell and spatial genomics.</h6>

    <p style="font-size: 10px;">Single-cell (sc)RNA-seq, together with RNA velocity and metabolic labeling, reveals cellular states and transitions at unprecedented resolution. Fully exploiting these data, however, requires kinetic models capable of unveiling governing regulatory functions. Here, we introduce an analytical framework dynamo (https://github.com/aristoteleo/dynamo-release), which infers absolute RNA velocity, reconstructs continuous vector fields that predict cell fates, employs differential geometry to extract underlying regulations, and ultimately predicts optimal reprogramming paths and perturbation outcomes. We highlight dynamo’s power to overcome fundamental limitations of conventional splicing-based RNA velocity analyses to enable accurate velocity estimations on a metabolically labeled human hematopoiesis scRNA-seq dataset. Furthermore, differential geometry analyses reveal mechanisms driving early megakaryocyte appearance and elucidate asymmetrical regulation within the PU.1-GATA1 circuit. Leveraging the least-action-path method, dynamo accurately predicts drivers of numerous hematopoietic transitions. Finally, <i> in silico</i> perturbations predict cell-fate diversions induced by gene perturbations. Dynamo, thus, represents an important step in advancing quantitative and predictive theories of cell-state transitions.</p>

  </div>
</div>