---
layout: paper
title: previous involved work
paper_title: Cicero predicts cis-regulatory DNA interactions from single-cell chromatin accessibility data
author_list: Hannah A Pliner, Jonathan S Packer, José L McFaline-Figueroa, Darren A Cusanovich, Riza M Daza, 
    Delasa Aghamirzaie, Sanjay Srivatsan, Xiaojie Qiu, Dana Jackson, Anna Minkina, Andrew C Adey, Frank J Steemers, 
    Jay Shendure+, Cole Trapnell+.
journal: Molecular Cell
doi: 10.1016/j.molcel.2018.06.044
year: 2018
image_url: /assets/images/papers/pliner-cicero.png
paper_alt: pliner-cicero Paper Image
rank: 6
---

Linking regulatory DNA elements to their target genes, which may be located hundreds of kilobases away, remains 
challenging. Here, we introduce Cicero, an algorithm that identifies co-accessible pairs of DNA elements using 
single-cell chromatin accessibility data and so connects regulatory elements to their putative target genes. We apply 
Cicero to investigate how dynamically accessible elements orchestrate gene regulation in differentiating myoblasts. 
Groups of Cicero-linked regulatory elements meet criteria of ‘‘chromatin hubs’’—they are enriched for physical 
proximity, interact with a common set of transcription factors, and undergo coordinated changes in histone marks that 
are predictive of changes in gene expression. Pseudotemporal analysis revealed that most DNA elements remain in 
chromatin hubs throughout differentiation. A subset of elements bound by MYOD1 in myoblasts exhibit early opening in a 
PBX1- and MEIS1-dependent manner. Our strategy can be applied to dissect the architecture, sequence determinants, 
and mechanisms of cis-regulation on a genome-wide scale.