---
layout: paper
title: "Read our work"
type: selected work
paper_title: Mapping transcriptomic vector fields of single cells
author_list: Xiaojie Qiu*+, Yan Zhang*, Jorge D Martin-Rufino, Chen Weng, Shayan Hosseinzadeh, Dian Yang, 
    Angela N Pogson, Marco Y Hein, Kyung Hoi Joseph Min, Li Wang, Emanuelle I Grody, Matthew J Shurtleff, 
    Ruoshi Yuan, Song Xu, Yian Ma, Joseph M Replogle, Eric S Lander, Spyros Darmanis, Ivet Bahar, 
    Vijay G Sankaran, Jianhua Xing+, Jonathan S Weissman+.
journal: Cell
doi: 10.1016/j.cell.2021.12.045
year: 2022
pdf_url: /assets/PDFs/qiu_dynamo.pdf
image_url: /assets/images/papers/dynamo.png
paper_alt: Dynamo Paper Image
rank: 1
---

Single-cell (sc)RNA-seq, together with RNA velocity and metabolic labeling, reveals cellular states and transitions at 
unprecedented resolution. Fully exploiting these data, however, requires kinetic models capable of unveiling governing 
regulatory functions. Here, we introduce an analytical framework dynamo (https://github.com/aristoteleo/dynamo-release), 
which infers absolute RNA velocity, reconstructs continuous vector fields that predict cell fates, employs differential 
geometry to extract underlying regulations, and ultimately predicts optimal reprogramming paths and perturbation 
outcomes. We highlight dynamoâ€™s power to overcome fundamental limitations of conventional splicing-based RNA velocity 
analyses to enable accurate velocity estimations on a metabolically labeled human hematopoiesis scRNA-seq dataset. 
Furthermore, differential geometry analyses reveal mechanisms driving early megakaryocyte appearance and elucidate 
asymmetrical regulation within the PU.1-GATA1 circuit. Leveraging the least-action-path method, dynamo accurately 
predicts drivers of numerous hematopoietic transitions. Finally, in silico perturbations predict cell-fate diversions 
induced by gene perturbations. Dynamo, thus, represents an important step in advancing quantitative
and predictive theories of cell-state transitions.