---
layout: paper
title: selected work
paper_title: Massively parallel and time-resolved RNA sequencing in single cells with scNT-seq
author_list: Qi Qiu*, Peng Hu*, Xiaojie Qiu, Kiya W Govek, Pablo G CÃ¡mara, Hao Wu+.
journal: Nature Methods
doi:
year: 2020
image_url: /assets/images/papers/qi_NM_2020.png
paper_alt: qi_NM_2023 Paper Image
rank: 8
---

Single-cell RNA sequencing offers snapshots of whole transcriptomes but obscures the temporal RNA dynamics. Here we 
present single-cell metabolically labeled new RNA tagging sequencing (scNT-seq), a method for massively parallel 
analysis of newly transcribed and pre-existing mRNAs from the same cell. This droplet microfluidics-based method 
enables high-throughput chemical conversion on barcoded beads, efficiently marking newly transcribed mRNAs with 
T-to-C substitutions. Using scNT-seq, we jointly profiled new and old transcriptomes in ~55,000 single cells. These 
data revealed time-resolved transcription factor activities and cell-state trajectories at the single-cell level in 
response to neuronal activation. We further determined rates of RNA biogenesis and decay to uncover RNA regulatory 
strategies during stepwise conversion between pluripotent and rare totipotent two-cell embryo (2C)-like stem cell 
states. Finally, integrating scNT-seq with genetic perturbation identifies DNA methylcytosine dioxygenase as an 
epigenetic barrier into the 2C-like cell state. Time-resolved single-cell transcriptomic analysis thus opens new 
lines of inquiry regarding cell-type-specific RNA regulatory mechanisms.



