---
layout: paper
title: selected work
paper_title: Spatiotemporally resolved transcriptomics reveals the subcellular RNA kinetic landscape
author_list: Jingyi Ren*, Haowen Zhou*, Hu Zeng*, Connie Kangni Wang, Jiahao Huang, Xiaojie Qiu, Xin Sui, Qiang Li, 
    Xunwei Wu, Zuwan Lin, Jennifer A Lo, Kamal Maher, Yichun He, Xin Tang, Judson Lam, Hongyu Chen, Brian Li, 
    David E Fisher, Jia Liu, Xiao Wang+.
journal: Nature Methods
doi:
year: 2023
image_url: /assets/images/papers/jingyi_tempomap.png
paper_alt: jingyi_tempomap Paper Image
rank: 10
---

Spatiotemporal regulation of the cellular transcriptome is crucial for proper protein expression and cellular function. 
However, the intricate subcellular dynamics of RNA remain obscured due to the limitations of existing transcriptomics 
methods. Here, we report TEMPOmap—a method that uncovers subcellular RNA profiles across time and space at the 
single-cell level. TEMPOmap integrates pulse-chase metabolic labeling with highly multiplexed three-dimensional in 
situ sequencing to simultaneously profile the age and location of individual RNA molecules. Using TEMPOmap, we 
constructed the subcellular RNA kinetic landscape in various human cells from transcription and translocation to 
degradation. Clustering analysis of RNA kinetic parameters across single cells revealed ‘kinetic gene clusters’ whose 
expression patterns were shaped by multistep kinetic sculpting. Importantly, these kinetic gene clusters are 
functionally segregated, suggesting that subcellular RNA kinetics are differentially regulated in a cell-state- and 
cell-type-dependent manner. Spatiotemporally resolved transcriptomics provides a gateway to uncovering new 
spatiotemporal gene regulation principles.