---
layout: "wrapper"
title: "READ OUR WORK"
---


<div class="container py-5">
  <div class="row">
    <div class="col-sm-12 col-md-5">
      <img
        id="gi-img"
        class="img-fluid my-2"
        src="assets/images/gi-image.png"
        alt="Cartoon image of a girl from Stanford typing at laptop"
      />
    </div>
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

    <div class="col-sm-12 col-md-7 mb-sm-4">
              <div class="media-body align-self-center">
                <div class="smallhead mb-1">
                    <strong><a href="/papers/Dynamo/" class="off">Mapping transcriptomic vector fields of single cells</a></strong>
                </div>
                <p class="note"><strong>Xiaojie Qiu</strong><sup>*</sup><sup>+</sup>, Yan Zhang<sup>*</sup>, Jorge D Martin-Rufino, Chen Weng, Shayan Hosseinzadeh, Dian Yang, Angela N Pogson, Marco Y Hein, Kyung Hoi Joseph Min, Li Wang, Emanuelle I Grody, Matthew J Shurtleff, Ruoshi Yuan, Song Xu, Yian Ma, Joseph M Replogle, Eric S Lander, Spyros Darmanis, Ivet Bahar, Vijay G Sankaran, Jianhua Xing<sup>+</sup>, Jonathan S Weissman<sup>+</sup> 2022.</p>
                <p class="note"><i>Cell</i></p>
                <p class="note"><sup>*</sup>Equal Contribution, <sup>+</sup>Corresponding Author</p>
            </div>
    </div>

    <div class="col-sm-12 col-md-5">
      <div class="glyphbox note">
    
        <div class="smallhead">PDF</div>
        <div class="pad-left note">
            <div class="smallspacer"></div>
            <i class="fa fa-file-text-o fa-fw"></i>
            <a class="off" href="/pdfs/papers/saunders-srivatsan-zf-crispants.pdf">saunders-srivatsan-zf-crispants.pdf</a>
        </div>
        <div class="bigspacer"></div>
        
        <div class="smallhead">DOI</div>
        <div class="pad-left note">
            <div class="smallspacer"></div>
            <i class="fa fa-external-link fa-fw"></i>
            <a class="off" href="http://dx.doi.org/10.1038/s41586-023-06720-2">10.1038/s41586-023-06720-2</a>
        </div>
        <div class="bigspacer"></div>
        
        <div class="smallhead">Preprint</div>
        <div class="pad-left note">
            <div class="smallspacer"></div>
            <i class="fa fa-external-link fa-fw"></i>
            <a class="off" href="http://dx.doi.org/10.1101/2022.08.04.502764v1">10.1101/2022.08.04.502764v1</a>
        </div>
        <div class="bigspacer"></div>
        
        <div class="smallhead">Source code</div>
        <div class="pad-left note">
            <div class="smallspacer"></div>
            <i class="fa fa-cog fa-fw"></i>
            <a class="off" href="https://github.com/cole-trapnell-lab/sdg-zfish">github.com/cole-trapnell-lab/sdg-zfish</a>
        </div>
        <div class="bigspacer"></div>

      </div>

    </div>

    <div class="col-sm-12 col-md-7 mb-sm-4">
        <h5>Abstract</h5>
        <p>Single-cell (sc)RNA-seq, together with RNA velocity and metabolic labeling, reveals cellular states and transitions at unprecedented resolution. Fully exploiting these data, however, requires kinetic models capable of unveiling governing regulatory functions. Here, we introduce an analytical framework <a href="https://github.com/aristoteleo/dynamo-release">dynamo</a>, which infers absolute RNA velocity, reconstructs continuous vector fields that predict cell fates, employs differential geometry to extract underlying regulations, and ultimately predicts optimal reprogramming paths and perturbation outcomes. We highlight dynamoâ€™s power to overcome fundamental limitations of conventional splicing-based RNA velocity analyses to enable accurate velocity estimations on a metabolically labeled human hematopoiesis scRNA-seq dataset. Furthermore, differential geometry analyses reveal mechanisms driving early megakaryocyte appearance and elucidate asymmetrical regulation within the PU.1-GATA1 circuit. Leveraging the least-action-path method, dynamo accurately predicts drivers of numerous hematopoietic transitions. Finally, in silico perturbations predict cell-fate diversions induced by gene perturbations. Dynamo, thus, represents an important step in advancing quantitative and predictive theories of cell-state transitions.</p>
    </div>

  </div>
</div>